gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -126,6 +127,7 @@ public final class InstantiatingGrpcChannelProvider implements TransportChannelP
   @Nullable private final Boolean allowNonDefaultServiceAccount;
   @VisibleForTesting final ImmutableMap<String, ?> directPathServiceConfig;
   @Nullable private final MtlsProvider mtlsProvider;
+  @Nullable private final ArrayList<String> allowedValues;
```

Comment:
Should this variable name be more specific? Reading it alone doesn't tell you where these values will be used.

Reply:
Thanks for calling this out! I updated this variable name to `allowedHardBoundTokenTypes`, WDYT?

Reply:
LGTM. Thanks.